etc/notes/CHANGES_5.0.0.md

```diff
@@ -184,17 +215,63 @@ collection.find(filterPredicate);
 
 **NOTE** As an experimental feature, these types can change at any time and are not recommended for production settings.
 
+### `bson-ext` support removed
+
+The `bson-ext` package will no longer automatically import and supplant the `bson` dependency.
+
+### Minimum supported Node version
```

Comment:
Wanna put this higher?

Reply:
I think bson-ext is more likely to surprise users than dropping an already EOL'd Node version.